pkg/controller/ptpoperatorconfig/ptpoperatorconfig_controller.go

```diff
@@ -201,6 +202,26 @@ func (r *ReconcilePtpOperatorConfig) syncLinuxptpDaemon(defaultCfg *ptpv1.PtpOpe
 	data.Data["Namespace"] = names.Namespace
 	data.Data["ReleaseVersion"] = os.Getenv("RELEASEVERSION")
 	data.Data["KubeRbacProxy"] = os.Getenv("KUBE_RBAC_PROXY_IMAGE")
+	data.Data["SideCar"] = os.Getenv("SIDECAR_EVENT_IMAGE")
+	// configure EventConfig
+	data.Data["EnableEventPublisher"] = "false"
+	data.Data["EventTransportHost"] = defaultCfg.Spec.EventConfig.TransportHost
```

Comment:
will this work if the default (not evenConfig exist)?

Reply:
Yes, the struct  will have empty data,  The default CRD does not have EventConfig and deploys without using EventConfig

Reply:
I made this data available via CLI 

$oc get ptpoperatorconfig
```
NAME      EVENT ENABLED   HOLDOVER TIMEOUT   MAX THRESHOLD   MIN THRESHOLD   EVENT TRANSPORT HOST
default   false           5                  100             -100            
```